SKFlatMaker/src/SKFlatMaker.cc

```diff
@@ -2622,15 +2629,68 @@ void SKFlatMaker::fillJet(const edm::Event &iEvent)
 
     bool tightJetID        = (NHF<0.90 && NEMF<0.90 && NumConst>1) &&            ((fabs(eta)<=2.4 && CHF>0 && CHM>0)              || fabs(eta)>2.4) && fabs(eta)<=2.7;
     bool tightLepVetoJetID = (NHF<0.90 && NEMF<0.90 && NumConst>1 && MUF<0.8) && ((fabs(eta)<=2.4 && CHF>0 && CHM>0 && CEMF<0.80) || fabs(eta)>2.4) && fabs(eta)<=2.7;
-    if(fabs(eta)>3.0){
-      tightJetID = (NEMF<0.90 && NHF>0.02 && NumNeutralParticles>10) && fabs(eta)>3.0;
-      tightLepVetoJetID = false; //FIXME default is false or true? following CAT2016
-    }
-    else if(fabs(eta)>2.7){
-      tightJetID = (NEMF>0.02 && NEMF<0.99) && (NumNeutralParticles>2) && abs(eta)>2.7 && abs(eta)<=3.0;
-      tightLepVetoJetID = false; //FIXME default is false or true? following CAT2016
+
+    if(DataYear==2016){
+      //=== 2016 jet IDs
+      //==== https://twiki.cern.ch/twiki/bin/view/CMS/JetID13TeVRun2016
+      
+      if(fabs(eta)>3.0){
+	tightJetID = (NEMF<0.90 && NumNeutralParticles>10 );
+	tightLepVetoJetID = (NEMF<0.90 && NumNeutralParticles>10 );
```

Comment:
TightLepVeto is not supported for |eta| > 2.7. Should we put false for this cases?

Reply:
I updated the PR as suggested